test/unit/email_rate_limit_test.rb

```diff
@@ -12,10 +12,12 @@ def seed_array
     valid_values          = seed_array.map { |n| n * multiplier }
     invalid_values        = last_clicked_days_ago.to_a - valid_values
     day_ago               = rand(last_clicked_days_ago)
-    clicked_ago           = valid_values.sample
-    assert EmailRateLimit.new(day_ago).now?(clicked_ago), "Expected  EmailRateLimit.new(#{day_ago}).now?(#{clicked_ago}) to be true, was not"
+    clicked_ago           = valid_values.reject { |int| (int % 7).zero? }.sample
```

Comment:
Can you walk me through why this helps? This is in the "daily email frequency" section and not in the "once a week" section.